### PR TITLE
Updating URLs on details page

### PR DESCRIPTION
**Pull request by ryelo**
Updated:
* playbook URL
* system URL should use inventory on current path (insights or rhcs)
* add conditional checking for beta on both

### CODE REVIEW

File: src/Utilities/urls.js [hunk 1]
```diff
@@ -28,9 +55,9 @@ export function buildIssueUrl (id) {
 
     switch (getIssuePrefix(id)) {
         case 'advisor':
-            return urijs(document.baseURI).segment('platform').segment('advisor').segment('actions').segment('by_id').segment(parts[1]).toString();
+            return urijs(document.baseURI).segment(`${isBeta()}insights`).segment('advisor').segment('actions').segment('by_id').segment(parts[1]).toString();
```
Comment (jharting) [inline src/Utilities/urls.js]: this should no longer use the advisor segment, right?

File: src/Utilities/urls.js [hunk 2]
```diff
@@ -1,9 +1,36 @@
...
+    return ('');
+}
+
+// Get the current group since we can be mounted at two urls, then add beta conditionally
+export function getGroup () {
+    const pathName = window.location.pathname.split('/');
+    pathName.shift();
+
+    if (pathName[0] === 'beta') {
+        pathName.shift();
```
Comment (jharting) [inline src/Utilities/urls.js]: @ryelo this probably should not be here

Comment (ryelo) [reply src/Utilities/urls.js]: Why not?

Comment (ryelo) [reply src/Utilities/urls.js]: Oh I see. Removing.

**Pull request merged**